debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -932,10 +932,15 @@ private Throwable handleRetries(final RetriableException e, final List<Map<Strin
         Throwable retryError = null;
         int maxRetries = getErrorsMaxRetries();
         LOGGER.info("Retriable exception thrown, connector will be restarted; errors.max.retries={}", maxRetries, e);
-        if (maxRetries < DEFAULT_ERROR_MAX_RETRIES) {
+        if (maxRetries == 0) {
             retryError = e;
         }
-        else if (maxRetries != 0) {
+        else if (maxRetries < DEFAULT_ERROR_MAX_RETRIES) {
+            LOGGER.warn("Setting {}={} is deprecated. To disable retries on connection errors, set {}=0", ERRORS_MAX_RETRIES.name(), ERRORS_MAX_RETRIES.name(),
```

Comment:
@nirolevy Hi, are you sure you provide correct arguments to the warn message?

Reply:
Fixed. Sorry, I re-worded in the last minute before pushing